### ISSUE TITLE
Learning rate - pre-training flair

### ISSUE THREAD

**Issue by pascalhuszar** [flairNLP/flair#1707]
Hey everybody,

i have question about the learning rate in pre-training a LM: Why is the learning rate set to 20? 
20 is relatively high, compared to other approaches like BERT.  And for example in training a flair LM for NER the learning rate is smaller than 1 and annealing is utilized. 
Is their a concept or idea behind this which i'm not aware of?

**Comment by alanakbik** [comment]
We generally found this to be a good learning rate for learning LMs with SGD. I think one reason is that the mini-batch size and sequence length in the default settings are quite high, which normally motivates a higher learning rate. Training the LM also involves an annealing scheme so the LR is reduced after some time.
